wpimath/src/main/native/include/frc/estimator/PoseEstimator.h

```diff
@@ -431,7 +434,7 @@ class WPILIB_DLLEXPORT PoseEstimator {
   // unless there have been no vision measurements after the last reset
   std::map<units::second_t, VisionUpdate> m_visionUpdates;
 
-  Pose2d m_poseEstimate;
+  Pose2d m_poseEstimate{};
```

Comment:
```suggestion
  Pose2d m_poseEstimate;
```

Reply:
For some reason I thought value-initialization was necessary to run the default constructor, guess I just misremembered.